service/worker/esanalyzer/domainWorkflowTypeCountWorkflow.go

```diff
@@ -160,6 +165,81 @@ func (w *Workflow) emitWorkflowTypeCountMetrics(ctx context.Context) error {
 	return nil
 }
 
+func (w *Workflow) getDomainWorkflowTypeCountPinotQuery(domainName string) (string, error) {
+	domain, err := w.analyzer.domainCache.GetDomain(domainName)
+	if err != nil {
+		return "", err
+	}
+	// exclude uninitialized workflow executions by checking whether record has start time field
+	// there's a "LIMIT 10" because in ES, Aggr clause by default returns the top 10 results
+	return fmt.Sprintf(`
+SELECT WorkflowType, COUNT(*) AS count
+FROM %s
+WHERE DomainID = '%s'
```

Comment:
nit: `%q`

Reply:
`%q` here might not work because double quote doesn't work in Pinot